tests/target/attrib.rs

```diff
@@ -1,5 +1,10 @@
 // rustfmt-wrap_comments: true
 // Test attributes and doc comments are preserved.
+#![doc(html_logo_url = "https://www.rust-lang.org/logos/rust-logo-128x128-blk-v2.png",
+       html_favicon_url = "https://doc.rust-lang.org/favicon.ico",
+       html_root_url = "https://doc.rust-lang.org/nightly/",
+       html_playground_url = "https://play.rust-lang.org/",
+       test(attr(deny(warnings))))]
```

Comment:
Thanks for the PR! I think that we should use block formatting rather than visual formatting though, e.g.,

```
#![doc(
    html_logo_url = "https://www.rust-lang.org/logos/rust-logo-128x128-blk-v2.png",
    html_favicon_url = "https://doc.rust-lang.org/favicon.ico",
    html_root_url = "https://doc.rust-lang.org/nightly/",
    html_playground_url = "https://play.rust-lang.org/",
    test(attr(deny(warnings))),
)]
```

(Although I think we need to follow macros and not insert a trailing comma if there is not one there already, since in the future it could be significant).

I think requiring all items in the list to be complex is probably to weak - e.g., imagine an attribute like the above example but with a trailing `0` - we'd still want to multi-line it, IMO. We could either have an `any` condition, rather than `all` or a 'more than half' condition (or something like that) or we could have something based on the overall length of the attribute - like always use vertical formatting if it goes multi-line.

Reply:
@nrc Thank you for your review. Actually the current implementation does the opposite of what I said at the top - it uses mixed layout only when all the items are 'simple'. My appologies for wrong description.

Reply:
I have updated the PR to use `indent_style` when formatting attributes, so that we use block indent style by deafult.